packages/core/src/actions/accounts/getAccount.ts

```diff
@@ -67,8 +67,8 @@ export function getAccount<
       } as const
     case 'connecting':
       return {
-        address: undefined,
-        connector: undefined,
+        address: data?.account,
```

Comment:
Noticed another weird behavior where the account & connector would be `undefined` if you are connecting to another connector.

Current behavior: If you are connected via Injected, then request connection to WalletConnect, the account + connector would be `undefined` (`connecting` status).
Expected behavior: If you are connected via Injected, then request connection to WalletConnect, the account + connector shouldn't change until you are either connected to the requested connector, or disconnected

Reply:
Makes sense. What is `connector` (below on line 71) set to in this case? For example, I connect with MM and now request a connection to WC: Is `address`, the currently connected MM address and `connector`, the pending WC connector?